modules/spring-config/src/main/java/org/mule/config/spring/parsers/specific/NameConstants.java

```diff
@@ -13,6 +15,11 @@
  */
 public class NameConstants
 {
+    public static final String MULE_NAMESPACE = "http://www.mulesoft.org/schema/mule/core";
+    public static final String MULE_PREFIX = "mule";
```

Comment:
these two are repeated on SchemaConstants. Remove it from there and change all uses to these references to these ones. Also verify there're no additional repetitions